tests/unit/rpc/RPCHelpersTests.cpp

```diff
@@ -547,6 +552,250 @@ TEST_F(RPCHelpersTest, ParseIssue)
     );
 }
 
+TEST_F(RPCHelpersTest, FetchBlobDoesNotExist)
```

Comment:
Test target is `fetchAndCheckAnyFlagsExists`. I suggest the name should indicate what is testing.